js/main.js

```diff
@@ -0,0 +1,17 @@
+const nav = document.querySelector('.nav-mobile');
+const navBtn = document.querySelector('.burger-btn');
+const allNavItems = document.querySelectorAll('.nav-mobile__item');
+
+const openNav = () => {
```

Comment:
If you use this function for both, opening and closing the menu, then it should have more appropriate name, like `toggleNav`

Reply:
I've changed it, thanks :)